### DISCUSSION TITLE
ELEGIR PALETA DE COLORES

### DISCUSSION THREAD

**Discussion by SaryNotfound** [Curcolor/COMMUNITY#3] [Ideas]
Estaré enviando paletas de colores para la página y me ayudarán a elegir una.
En el pie de las imagenes sale el numero de la paleta para que escojan

![paleta 1](https://github.com/user-attachments/assets/db222d11-138a-4ef4-83fe-e144602d6626)
 
![paleta 2](https://github.com/user-attachments/assets/82b7bd8d-6ca6-4736-8475-198ec2b5ea29)

![paleta 3](https://github.com/user-attachments/assets/ab15ac0e-fb35-497a-9fd7-6f8f2a924db5)

![paleta 4](https://github.com/user-attachments/assets/40b01764-6441-4874-b15a-cd4b61003d6f)

**Comment by Curcolor** [comment]
Pues teniendo en cuenta las discusiones anteriores en WhatsApp,  y por propia decisión me quedo con la **paleta 1**  , Gracias @SaryNotfound por el buen trabajo dando las opciones.
